templates/default/activiti/db.properties.erb

```diff
@@ -0,0 +1,5 @@
+db=
```

Comment:
Why not generating this file, like we do with alfresco-global.properties (using artifact-deployer)?